lib/susedistribution.pm

```diff
@@ -518,6 +518,16 @@ sub init_consoles {
                 serial => 'rm -f /dev/sshserial; mkfifo /dev/sshserial; chmod 666 /dev/sshserial; while true; do cat /dev/sshserial; done',
                 gui => 1
             });
+        $self->add_console(
+            'root-ssh-virt',
+            'ssh-xterm',
+            {
+                hostname => get_required_var('SUT_IP'),
+                password => $testapi::password,
+                username => 'root',
+                serial => 'rm -f /dev/sshserial; mkfifo /dev/sshserial; chmod 666 /dev/sshserial; while true; do cat /dev/sshserial; done',
```

Comment:
In this way, the two consoles `root-ssh` and `root-ssh-virt` will share the same /dev/sshserial. It will be problematic when both consoles are actively used at the same time. Suggest to use different device name .